espresso-support/demo/src/androidTest/java/com/novoda/movies/DemoMainActivityTest.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.movies;
+
+import android.support.test.filters.LargeTest;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class DemoMainActivityTest {
+
+    @Rule
+    public ActivityTestRule<DemoMainActivity> activityRule = new ActivityTestRule<>(DemoMainActivity.class);
+
+    @Test
+    public void testActivityNotNull() {
+        if (activityRule.getActivity() == null) {
```

Comment:
this test is just to assert that the `extras` artifact is not necessary if you're not using `ViewActivityRule`. If you comment out that dependency, then this test will run perfectly fine. The MovieItemViewTest will _compile_ fine, but will fail at runtime because it cannot resolve `ViewActivity` which is in the `extras` artifact.